src/core/fonts.js

```diff
@@ -243,6 +243,12 @@ function getFontType(type, subtype) {
   }
 }
 
+function getStandardFontName(name) {
+  const fontName = name.replace(/[,_]/g, "-").replace(/\s/g, "");
```

Comment:
Since you're using the same regular expression at https://github.com/mozilla/pdf.js/blob/00b4f86db301546d21d5dc79c61e2a9a829667a4/src/core/fonts.js#L1319
can you please extract the regexp-part to a new constant, used by both call-sites, to avoid these getting out of sync?